platform/android/display_server_android.h

```diff
@@ -205,11 +205,11 @@ class DisplayServerAndroid : public DisplayServer {
 	void process_joy_event(JoypadEvent p_event);
 	void process_key_event(int p_keycode, int p_scancode, int p_unicode_char, bool p_pressed);
 
-	virtual void cursor_set_shape(CursorShape p_shape);
-	virtual CursorShape cursor_get_shape() const;
+	virtual void cursor_set_shape(CursorShape p_shape) override;
+	virtual CursorShape cursor_get_shape() const override;
 
-	void mouse_set_mode(MouseMode p_mode);
-	MouseMode mouse_get_mode() const;
+	void mouse_set_mode(MouseMode p_mode) override;
+	MouseMode mouse_get_mode() const override;
```

Comment:
Adding `override` here is correct, but then it needs to be done for all other overridden methods in the class too, otherwise that raises a warning: https://github.com/godotengine/godot/pull/50077/checks?check_run_id=2973352976

That being said other implementations don't use `override` either so I wouldn't add it in this PR, it needs to be done for all classes if it's wanted.

Reply:
was way past you, already added to everything in this class

Reply:
Yeah but now this class is inconsistent with other `DisplayServer` implementations (at least Windows and Linux).

Reply:
no problem, will remove it

Reply:
removed

Reply:
Ill add that to all other display servers in other time